src/games/seabattle/mocks.ts

```diff
@@ -11,6 +12,7 @@ export const VALID_SETUP_FIRST_PLAYER: IShip[] = [
       { x: 4, y: 0 },
     ],
     sunk: true,
+    id: shortid.generate(),
```

Comment:
These are mocks for testing, so use deterministic IDs like "foo" and "bar" instead of random ids

Reply:
Good idea